src/Common/src/System/IO/DelegatingStream.cs

```diff
@@ -172,6 +172,11 @@ public override void EndWrite(IAsyncResult asyncResult)
             _innerStream.EndWrite(asyncResult);
         }
 
+        public override void CopyTo(Stream destination, int bufferSize)
+        {
+            _innerStream.CopyTo(destination, bufferSize);
+        }
```

Comment:
(That said, this use case also isn't particularly significant: the way you get such a stream is via async APIs: you're less likely at that point to switch to using sync ones, and if you do, you're likely already paying a perf penalty.)